camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/test/lite/CampPlatformWithJustBrooklynMgmt.java

```diff
@@ -30,7 +30,9 @@
 
     public CampPlatformWithJustBrooklynMgmt(ManagementContext mgmt) {
         this.mgmt = mgmt;
+        // Deprecated in 0.11.0. Add to release notes and remove in next release.
```

Comment:
maybe tweak the comment to clarify that it is the `use of getConfig().put() on the following line` that is deprecated in favour of the use of the scratchpad. (and same goes for similar comments elsewhere)